src/main/java/org/mule/extension/db/internal/domain/executor/BulkUpdateExecutor.java

```diff
@@ -98,4 +89,30 @@ public Object execute(DbConnection connection, Query query, List<List<QueryParam
       statement.close();
     }
   }
+
+  private void logBulkUpdateErrorInfo(int[] updateCounts, int batchCount) {
+    int successfulOperations, failedOperations, noInfoAvailable;
+    successfulOperations = 0;
+    failedOperations = 0;
+    noInfoAvailable = 0;
+    for (int i = 0; i < updateCounts.length; i++) {
+      if (updateCounts[i] >= 0) {
+        successfulOperations++;
+        LOGGER.debug("BULK UPDATE SUCCESSFUL OPERATION PERFORMED: %d AFFECTED ROWS", updateCounts[i]);
+      } else if (updateCounts[i] == Statement.SUCCESS_NO_INFO) {
+        noInfoAvailable++;
+        LOGGER.debug("BULK UPDATE SUCCESSFUL OPERATION PERFORMED: NO INFO AVAILABLE ON AFFECTED ROW COUNT");
+      } else if (updateCounts[i] == Statement.EXECUTE_FAILED) {
+        failedOperations++;
+        LOGGER.debug("BULK UPDATE OPERATION FAILED:  %d AFFECTED ROWS", updateCounts[i]);
+      }
+    }
```

Comment:
Message could be improved showing the number of statement that results belong to.